Frends.SMTP.SendEmail/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.1] - 2022-11-23
```

Comment:
No need to update version, since no new implementations or fixes have been made to the task implementation.